GiniVision/Classes/CameraViewController.swift

```diff
@@ -84,7 +93,10 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
         return UIImageNamedPreferred(named: "cameraCaptureButtonActive")
     }
     fileprivate var cameraOverlayImage: UIImage? {
-        return UIImageNamedPreferred(named: "cameraOverlay")
+        guard let image = UIImageNamedPreferred(named: "cameraOverlay"), let cgImage = image.cgImage else{
```

Comment:
I can create higher resolution images for the iPad from the photoshop assets file (as I did for Android). 

I prefer continuing using images to be consistent. Clients would only need to provide new iPad images, which can be expected when updating to support iPads.